code/game/sound.dm

```diff
@@ -177,10 +152,6 @@
 			var/area/A = get_area(src)
 			sound_to_use.environment = A.sound_environment
 
-		if(use_reverb && sound_to_use.environment != SOUND_ENVIRONMENT_NONE) //We have reverb, reset our echo setting
-			sound_to_use.echo[3] = 0 //Room setting, 0 means normal reverb
-			sound_to_use.echo[4] = 0 //RoomHF setting, 0 means normal reverb.
```

Comment:
I believe this code is still necessary ultimately, at least
```dm
if(!use_reverb || sound_to_use.environment == SOUND_ENVIRONMENT_NONE)
	sound_to_use.echo = new /list(18)
	sound_to_use.echo[3] = -10000
	sound_to_use.echo[4] = -10000
```
Otherwise, anything that uses `use_reverb` will reverb. Which will probably be annoying (stuff like 'heartbeats' and 'antag stingers' use no reverb).

This does mean everything with `environment == SOUND_ENVIRONMENT_NONE` will be downmixed, though.